test/kubemark/resources/manifests/kube-addon-manager.yaml

```diff
@@ -12,8 +12,7 @@ spec:
     # When updating version also bump it in:
     # - cluster/images/hyperkube/static-pods/addon-manager-singlenode.json
     # - cluster/images/hyperkube/static-pods/addon-manager-multinode.json
-    # - cluster/gce/coreos/kube-manifests/kube-addon-manager.yaml
```

Comment:
@gyliu513 Removed the pointers in `test/kubemark/resources/manifests/kube-addon-manager.yaml` and updated the references in `cluster/saltbase/salt/kube-addons/kube-addon-manager.yaml`, which is expected to be the entry point when bumping up addon-manager.

Yeah these comments are messy, but it would be better to have it. At least people will know they may need to `grep`. I hope we will later have pre-submit checks to ensure files are in-synced (by putting specific lint comments in codes).

@jbeda Sorry for the change after lgtm...